File: app/src/main/java/com/anadol/mindpalace/domain/formatters/PercentFormatter.java
package com.anadol.mindpalace.domain.formatters;

import com.github.mikephil.charting.formatter.ValueFormatter;

public class PercentFormatter extends ValueFormatter {

    @Override
    public String getFormattedValue(float value) {
        return Math.round(value) + "%";
    }
}

File: app/src/main/java/com/anadol/mindpalace/domain/formatters/PercentFormatter.kt
package com.anadol.mindpalace.domain.formatters

import com.github.mikephil.charting.formatter.ValueFormatter
import kotlin.math.roundToInt

class PercentFormatter : ValueFormatter() {

    override fun getFormattedValue(value: Float): String = "${value.roundToInt()} %"

}